### PR TITLE
Add styles based on the prototype

### PR DESCRIPTION
**Pull request by vegeorgi**
[prototype](http://ux.telerik.com/wireframes/NS%20Starter%20Kit/start.html#g=1&p=master_detail)

### CODE REVIEW

**Comment by manoldonev** [comment]
"Automatic transmissi..." is shown on the master list -- we need to get rid of the ellipsis.

**Comment by manoldonev** [comment]
The modal page (e.g. when customizing car class) looks a bit strange on iOS (long trailing gray band)?

**Comment by manoldonev** [comment]
Also, I find the modal dialog item options (on Android) a bit too small for fat fingers :)

File: cars/car-detail-edit-page/car-detail-edit-view-model.ts [hunk 1]
```diff
@@ -6,9 +6,8 @@ import firebase = require("nativescript-plugin-firebase");
 
 import { Car } from "../shared/car-model";
 import { RoundingValueConverter } from "./RoundingValueConverter";
```
Comment (manoldonev) [inline cars/car-detail-edit-page/car-detail-edit-view-model.ts]: Should be `from "./roundingValueConverter";` and `from ./visibilityValueConverter` (lower case). It seems I have erroneously committed the rounding value converter with upper case in the first place (for TypeScript).

Can you fix them?

Comment (vegeorgi) [inline cars/car-detail-edit-page/car-detail-edit-view-model.ts]: 👍

**Pull request merged**